src/main/java/com/gooddata/project/Role.java

```diff
@@ -0,0 +1,101 @@
+package com.gooddata.project;
+
+import com.gooddata.md.Meta;
+import com.gooddata.util.BooleanStringDeserializer;
+import com.gooddata.util.BooleanStringSerializer;
+import org.codehaus.jackson.annotate.JsonCreator;
+import org.codehaus.jackson.annotate.JsonIgnore;
+import org.codehaus.jackson.annotate.JsonIgnoreProperties;
+import org.codehaus.jackson.annotate.JsonProperty;
+import org.codehaus.jackson.annotate.JsonTypeInfo;
+import org.codehaus.jackson.annotate.JsonTypeName;
+import org.codehaus.jackson.map.annotate.JsonDeserialize;
+import org.codehaus.jackson.map.annotate.JsonSerialize;
+import org.springframework.web.util.UriTemplate;
+
+import java.util.Collections;
+import java.util.HashMap;
+import java.util.Map;
+
+/**
+ * Project Role
+ */
+@JsonTypeInfo(include = JsonTypeInfo.As.WRAPPER_OBJECT, use = JsonTypeInfo.Id.NAME)
+@JsonTypeName("projectRole")
+@JsonIgnoreProperties(ignoreUnknown = true)
+@JsonSerialize(include = JsonSerialize.Inclusion.NON_NULL)
+public class Role {
+    public static final String URI = "/gdc/projects/{projectId}/roles/{roleId}";
+    public static final UriTemplate TEMPLATE = new UriTemplate(URI);
+
+    private static final String SELF_LINK = "self";
+
+    @JsonDeserialize(contentUsing = BooleanStringDeserializer.class)
+    @JsonSerialize(contentUsing = BooleanStringSerializer.class)
+    private final Map<String, Boolean> permissions;
+
+    private Meta meta;
+
+    private final Map<String,String> links;
+
+    @JsonCreator
+    Role(@JsonProperty("permissions") final Map<String, Boolean> permissions,
+         @JsonProperty("meta") final Meta meta,
+         @JsonProperty("links") final Map<String, String> links) {
+        this.permissions = permissions == null ? new HashMap<String, Boolean>() : permissions;
+        this.meta = meta == null ? new Meta(null) : meta;
+        this.links = links == null ? new HashMap<String, String>() : links;
+    }
+
+    public Map<String, Boolean> getPermissions() {
```

Comment:
definitely not, but the `Map` is a bit weird. What about `Set<String> getEnabledPermissions()` and `Set<String> getAvailablePermissions()`?